spec/models/chess_piece_spec.rb

```diff
@@ -140,5 +140,19 @@
     end
   end
 
+  describe "#capture?" do
+    context "given a black piece at [4,4] and a white piece at [5,5]" do
+      let(:game) {Game.create}
+      let(:piece) {ChessPiece.create(position_x: 4, position_y: 4, game: game, color: "black")}
+      let(:second_piece) {ChessPiece.create(position_x: 5, position_y: 5, game: game, color: "white")}
+      it "will return true if piece.color != second_piece.color" do
+        expect(piece.color != second_piece.color).to be_truthy
```

Comment:
let's think about what this is testing. You're testing that `piece.color != second_piece.color` which will never be false because of the way your tests are setup.

so here's what it's saying right now.

```ruby
piece = ChessPiece.create(position_x: 4, position_y: 4, game: game, color: "black")
second_piece = ChessPiece.create(position_x: 5, position_y: 5, game: game, color: "white")

expect(piece.color != second_piece.color).to be_truthy
# translation: I expect the first piece and second piece to have different colors
```

Which is not exactly what we want.

We want to tests

> `capture` piece at `endpoint_x`, `endpoint_y` if it is not the same color

try to figure out how to write that sentence as a test. feel free to ask teammates for help =)